win32ss/gdi/eng/surface.c

```diff
@@ -171,9 +175,9 @@ SURFACE_AllocSurface(
             /* Make sure the buffer is large enough */
             if (cjBufSize < cjBits)
             {
-                DPRINT1("Buffer is too small, required: %lu, got %lu\n",
-                        cjBits, cjBufSize);
-                return NULL;
+                DPRINT("Buffer is too small, required: %lu, got %lu\n",
+                    cjBits, cjBufSize);
```

Comment:
Indentation broken.

Also, what kind of values (required vs. provided) do you get when testing with Office 2000 and the Hoyle Cards?

Reply:
@HBelusca, Prior to my changes, the output was like the following (taken from and actual run):
```
(win32ss/gdi/eng/surface.c:175) Buffer is too small, required: 4224, got 3944
```
All of the example with problems showed a smaller number for 'got' than what was shown for 'required'.